.github/workflows/ci.yaml

```diff
@@ -69,3 +69,24 @@ jobs:
       # - name: Setup SSH if failure
       #   if: failure()
       #   uses: lhotari/action-upterm@v1
+
+  check-contracts-changes:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Check for changes
+        uses: dorny/paths-filter@v3
+        id: changes
+        with:
+          filters: |
+            contracts_dir: 
+              - 'contracts/v2.6.2/**'
+
+      - name: Install scarb
+        if: steps.changes.outputs.contracts_dir == 'true'
+        run: curl --proto '=https' --tlsv1.2 -sSf https://docs.swmansion.com/scarb/install.sh | sh
```

Comment:
I think we could have a mismatch of `scarb` version here since it will installed the latest stable version.

In melos, scarb installation is made with `asdf` so version is managed with `.tool-versions` file.